jormungandr-lib/Cargo.toml

```diff
@@ -29,7 +29,6 @@ hex = "0.4"
 bech32 = "0.8"
 base64 = "0.13.0"
 http = "0.2.2"
-eccoxide = { git = "https://github.com/eugene-babichenko/eccoxide.git", branch = "fast-u64-scalar-mul", features = ["fast-u64-scalar-mul"], optional = true }
```

Comment:
`eccoxide` was never a direct dependency of this crate, but rather a hack to work around the current build process https://github.com/input-output-hk/jormungandr/pull/3184. We should switch away from that `eccoxide` version though, as I moved that improvement directly in chain-crypto IIRC.

Reply:
oh, I remember now. As far as I understand, we still need to remove it from here then, right?

Reply:
yes, and actually it was re-inserted in chain-libs somehow, https://github.com/input-output-hk/chain-libs/pull/714 will fix it